### ISSUE TITLE
"change Puppet Master" option exists but it doesn't work

### ISSUE THREAD

**Issue by Ron-Lavi** [theforeman/foreman_puppet#330]
From https://projects.theforeman.org/issues/35949

Description of problem:
Web UI, Hosts, All Hosts, Select target hosts, Down Arrow next to 'Select Action', change Puppet Master and Select Clear Puppet CA will prompt a dialog and spin forever because of the missing path error.

How reproducible:
Always

Additional info:
From Hao's notes:
~~~
Even before disabling the foreman_puppet plugin, "change Puppet Master" option exists but it doesn't work. After pressing the "change Puppet Master", it will prompt a dialog and spin forever because of the missing path error. Need to raise a bugzilla for this if not any is raised yet.

production.log error
```
2022-12-16T11:02:52 [I|app|cc3a5bd2] Rendering hosts/select_multiple_puppet_proxy.html.erb within layouts/application
2022-12-16T11:02:52 [I|app|cc3a5bd2] Rendered hosts/_selected_hosts.html.erb (Duration: 16.6ms | Allocations: 5533)
2022-12-16T11:02:52 [I|app|cc3a5bd2] Rendered hosts/select_multiple_puppet_proxy.html.erb within layouts/application (Duration: 479.7ms | Allocations: 46616)
2022-12-16T11:02:52 [W|app|cc3a5bd2] undefined method `update_multiple_puppet_proxy_hosts_path' for #<#<Class:0x000000001c5e07d8>:0x000000001bbe3c58>
cc3a5bd2 | Did you mean? update_multiple_puppet_ca_proxy_hosts_path
cc3a5bd2 | update_multiple_puppet_ca_proxy_hosts_url
cc3a5bd2 | update_multiple_openscap_proxy_hosts_path
cc3a5bd2 | update_multiple_owner_hosts_path
cc3a5bd2 | update_multiple_power_state_hosts_path
cc3a5bd2 | update_multiple_hostgroup_hosts_path
cc3a5bd2 | update_multiple_parameters_hosts_path
cc3a5bd2 | update_multiple_location_hosts_path
```

**Comment by Ron-Lavi** [comment]
Hey @sbernhard! not yet, but I am planning to get to it soon

**Comment by nadjaheitmann** [comment]
Fixed by #344 . We should cherry-pick this to 4.1-stable if it the foreman fix gets cherry-picked as well. Would make sense from my point of view.
